crates/ide_completion/src/render.rs

```diff
@@ -987,4 +1026,160 @@ fn main() {
             "#]],
         )
     }
+
+    #[test]
+    fn score_fn_type_and_name_match() {
```

Comment:
It's better to avoid full verbose check. If we are testing scores, we should check only the scores. `check_relevance` function does that.

Reply:
Ah yes thanks for pointing this out! I've converted to `check_relevance` as suggested.